src/persistance/metadataWriter.ts

```diff
@@ -0,0 +1,20 @@
+import { Metadata } from "../metadata/MetadataGenerable";
+import { META_DATA_LOCATION } from "../constants";
+
+export default class MetadataWriter {
+  private _fs: any;
+
+  constructor(fs: any) {
+    this._fs = fs;
+  }
+
+  writeMetadata(metaData: Metadata) {
+    const fileName: string = metaData.type + ".dat";
+    const fileLocation = META_DATA_LOCATION + "/" + fileName;
+    this._fs.writeFileSync(
+      fileLocation,
+      JSON.stringify(metaData.data),
+      "utf-8"
+    );
+  }
+}
```

Comment:
Please change the file name to be all small and hyphenated for js/ts files. [Google's guide](https://google.github.io/styleguide/jsguide.html#file-name).

Reply:
Done.